_custom_build/backend.py

```diff
@@ -0,0 +1,49 @@
+from setuptools import build_meta as _orig
+from packaging import version as _version
+from packaging.tags import sys_tags as _sys_tags
+from skbuild.exceptions import SKBuildError as _SKBuildError
+from skbuild.cmaker import get_cmake_version as _get_cmake_version
+import subprocess as _subprocess
+import platform as _platform
+
+prepare_metadata_for_build_wheel = _orig.prepare_metadata_for_build_wheel
+build_wheel = _orig.build_wheel
+build_sdist = _orig.build_sdist
+get_requires_for_build_sdist = _orig.get_requires_for_build_sdist
+
+def _cmake_required():
+    try:
+        version = _version.parse(_get_cmake_version())
+        if version in {_version.parse("3.17.0"), _version.parse("3.17.1")}
```

Comment:
I am not sure why this was excluded. Was cmake broken, or was the python package for cmake broken in these versions?